src/ngraph/runtime/cpu/cpu_builder.hpp

```diff
@@ -29,6 +29,134 @@
+        SELECT_RANK(KV, int8_t, R, K);                                                             \
+    }                                                                                              \
+    else if (ET == element::i16)                                                                   \
+    {                                                                                              \
+        SELECT_RANK(KV, int16_t, R, K);                                                            \
+    }                                                                                              \
+    else if (ET == element::i32)                                                                   \
+    {                                                                                              \
+        SELECT_RANK(KV, int32_t, R, K);                                                            \
+    }                                                                                              \
+    else if (ET == element::i64)                                                                   \
+    {                                                                                              \
+        SELECT_RANK(KV, int64_t, R, K);                                                            \
+    }                                                                                              \
+    else if (ET == element::u8)                                                                    \
+    {                                                                                              \
+        SELECT_RANK(KV, uint8_t, R, K);                                                            \
+    }                                                                                              \
+    else if (ET == element::u16)                                                                   \
+    {                                                                                              \
+        SELECT_RANK(KV, uint16_t, R, K);                                                           \
+    }                                                                                              \
+    else if (ET == element::u32)                                                                   \
+    {                                                                                              \
+        SELECT_RANK(KV, uint32_t, R, K);                                                           \
+    }                                                                                              \
+    else if (ET == element::u64)                                                                   \
+    {                                                                                              \
+        SELECT_RANK(KV, uint64_t, R, K);                                                           \
+    }
+
```

Comment:
i'm assuming that this will be used in future PRs?

Reply:
Yes, just wanted to get the header change in earlier with the refactor